server/db/helper.js

```diff
@@ -67,9 +67,18 @@ const getCourse = (id) => {
     });
 };
 
+const getUserBadges = (userId) => UserBadge.findAll({ 
+  where: {
+    id: userId,
+  },
+  
+  attributes: ['idBadge'],
+});
```

Comment:
You also want the helper to then search the Badge table and return the relevant info for each badge that was found previously (sequelize lets you pass in arrays as search conditions like - where: { id : arrayVariable })

Reply:
Agree. So the helper 'findsall' matching badge Ids, (which you are doing now) and then for each badge the users has, the helper searches the badge table.  and return the relevant data. 
From the badge table,  I'd grab the name, icornURl, and description (experience Points seem unnecessary to me).